md/single-sign-on-with-cas.md

```diff
@@ -153,6 +165,12 @@ Bonita BPM uses the CAS LoginModule in the JASIG implementation, so see the CAS
 
 1. For each tenant, edit `authenticationManager-config.properties` to enable the CASRemoteAuthenticationManager and its properties.
 Edit the `authenticationManager-config.properties` located in `platform_conf/initial/tenant_template_portal` for not initialized platform or `platform_conf/current/tenant_template_portal` and `platform_conf/current/tenants/[TENANT_ID]/tenant_portal/`.
+::: info
+If the platform has already been initialized, every update to the configuration files under `setup/platform_conf/current` must be done using the `setup` tool:  
+- setup pull  
```

Comment:
you used quotes above (`setup pull`)
It would be better to either keep them everywhere or remove them everywhere